iptables/iptables.go

```diff
@@ -39,23 +39,41 @@ func (e *Error) Error() string {
 	return fmt.Sprintf("exit status %v: %v", e.ExitStatus(), e.msg)
 }
 
+// Protocol to differentiate between IPv4 and IPv6
+type Protocol byte
+
+const (
+	ProtocolIPv4 Protocol = iota
+	ProtocolIPv6
+)
+
 type IPTables struct {
 	path     string
+	proto    Protocol
 	hasCheck bool
 	hasWait  bool
 }
 
+// Creates a new IPTables.
```

Comment:
Here and further below, please follow the complete-sentence suggestion from https://blog.golang.org/godoc-documenting-go-code, e.g. `New creates a...`.